src/pyChatGPT/pyChatGPT.py

```diff
@@ -315,6 +316,29 @@ def __ensure_cf(self, retry: int = 0) -> None:
         self.__verbose_print('Closing tab')
         self.driver.close()
         self.driver.switch_to.window(original_window)
+        
+    def __click_try_again(self) -> WebElement:
+        self.__verbose_print('Click try again button')
+        try_again = self.driver.find_elements(By.XPATH, '//*[@id="__next"]/div[2]/div[1]/main/div[2]/form/div/div[1]/button')[-1]
+        try_again.click()
+
+        request = self.driver.find_elements(
+            By.XPATH, "//div[starts-with(@class, 'request-:')]"
```

Comment:
I think the xpath can be changed to something more simple like `//button[text()="Try again"]`.